tests/queries/0_stateless/02346_text_index_direct_read_bug90802.sql

```diff
@@ -0,0 +1,34 @@
+-- Test cases for optimization for like using text index
+
```

Comment:
Please add a couple of lines explaining the issue

Reply:
Hi @Ergus ，I’ve fixed this in the latest commit.